runtime/flags/build.rs

```diff
@@ -0,0 +1,130 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+#![allow(clippy::disallowed_methods)]
+
+#[derive(serde::Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+enum JsonEntryKind {
+  Cli,
+  Runtime,
+}
+
+#[derive(serde::Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+struct JsonEntry {
+  name: String,
+  text: String,
+  show_in_help: bool,
+  #[serde(default)]
+  env_var: Option<String>,
+  kind: JsonEntryKind,
+}
+
+fn main() {
+  let crate_dir = std::path::PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+  println!(
+    "cargo:rerun-if-changed={}",
+    crate_dir.join("flags.json").display()
+  );
+
+  let mut entries: Vec<JsonEntry> =
+    serde_json::from_str(include_str!("./flags.json")).unwrap();
+
+  entries.sort_by(|a, b| a.name.cmp(&b.name));
+
+  let mut rust_list = String::from(
```

Comment:
In comparison to https://github.com/denoland/deno/pull/28888 I opted to codegen these files into this directory - so they can be introspected easily.